File: PP2/pages/api/code/temp/main.hs
-- Import the necessary module for reading input
import System.IO

main :: IO ()
main = do
    -- Prompt the user for input
    putStrLn "Enter something: "
    
    -- Read the user input
    userInput <- getLine
    
    -- Print the entered input
    putStrLn ("You entered: " ++ userInput)

File: PP2/pages/api/code/temp/Main.go
package main

import (
	"fmt"
	"bufio"
	"os"
)

func main() {
	// Read input from the user
	reader := bufio.NewReader(os.Stdin)
	input, _ := reader.ReadString('\n')

	// Print the input
	fmt.Printf("You entered: %s", input)
}
